lib/notes.test.js

```diff
@@ -1,2 +1,24 @@
-const {add,  expect} = require('./notes');
+const Notes = require('./Notes');
 
+describe('Notes class', () => {
+
+    
+    it('retruns a note', () => {
+
+        const notes = {
+            type: 'add',
+            payload: 'mytext'
+
+        }
+        const results = Notes.exicute(notes);
+        expect(results).toEqual({
+        id: expect.any(Number),
+        text: 'mytext'})
+    
+    });
+
+  
+
+
+
+  });
```

Comment:
files should end with a newline. you can configure vs code to add it for you automatically